lighthouse-core/lib/page-functions.js

```diff
@@ -384,11 +392,11 @@ function isPositionFixed(element) {
  * Generate a human-readable label for the given element, based on end-user facing
  * strings like the innerText or alt attribute.
  * Falls back to the tagName if no useful label is found.
- * @param {Element} node
+ * @param {Element} element
  * @return {string}
  */
-function getNodeLabel(node) {
-  // Inline so that audits that import getNodeLabel don't
+function getNodeLabel(element) {
+  // Inline so that audits that import getNodeLabel doesn't
```

Comment:
```suggestion
  // Inline so that audits that import getNodeLabel don't
```
yeah that's not right